docs/CNAME

```diff
@@ -1 +0,0 @@
-charts.appuio.ch
```

Comment:
Uh, not sure if this file should be deleted?

Reply:
All the compiled stuff has been moved to the `gh-pages` branch; the `CNAME` file should already be there